test/spec/support/container_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ContainerHelper
+
+  # Checks if a deployed container exists with given name
+  # @param [String] name of the container to check
+  def container_exist?(name)
+    k = run "kontena container ls"
```

Comment:
good idea to use `-q`, pushed fix for that.

we know in this case at least that there's no similarly named container so should be safe.